internal/pkg/build/sources/conveyorPacker_library.go

```diff
@@ -70,14 +81,47 @@ func (cp *LibraryConveyorPacker) Get(b *types.Bundle) (err error) {
 		}
 	}
 
+	if !cp.AllowUnauthenticated {
+		// check if the base container is signed
+		imageSigned, err := signing.IsSigned(imagePath, "https://keys.sylabs.io", 0, false, cp.AuthToken, cp.LocalVerify, true)
```

Comment:
the URL should not be hardcoded here. Pull has an option to specify this and there are at least another two places where this URL already exists.